hip/test/components/cooperative_groups.hip.cpp

```diff
@@ -262,4 +267,61 @@ TEST_F(CooperativeGroups, SubwarpBallot) { test(cg_subwarp_ballot); }
 TEST_F(CooperativeGroups, SubwarpBallot2) { test_subwarp(cg_subwarp_ballot); }
 
 
+template <typename ValueType>
+__global__ void cg_shuffle_sum(const int num, ValueType *__restrict__ value)
+{
+    auto group =
+        group::tiled_partition<config::warp_size>(group::this_thread_block());
+    for (int ind = 0; ind < num; ind++) {
+        value[group.thread_rank()] += group.shfl(value[ind], ind);
+    }
+}
+
+TEST_F(CooperativeGroups, ShuffleSumDouble)
```

Comment:
For some reason, this file only has a single empty line between the tests (`TEST_F`).
In pretty much all other tests I was looking into, we have 2 empty lines to separate the tests. Could you also do that for this file?

Reply:
Sure, they are done.